docs/languages/powershell.md

```diff
@@ -172,3 +172,34 @@ CodeLens function reference support shows the number of times a function is refe
 ## Extension FAQ page
 
 Check out the FAQ page on the [PowerShell extensions Wiki](https://github.com/PowerShell/vscode-powershell/wiki/FAQ)
+
+## Types.ps1xml and Format.ps1xml files
+
+You can get IntelliSense features when authoring `Types.ps1xml` and `Format.ps1xml` files by installing the [XML extension by RedHat](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml).
```

Comment:
Link to https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_format.ps1xml?view=powershell-6 and https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_types.ps1xml?view=powershell-6

Reply:
Maybe you should also include a sentence or 2 in this PR explaining what these files are.

```
## Types.ps1xml and Format.ps1xml files

`ps1xml` files are PowerShell's way to extend the type system and define output formatting. For more information on these files, please refer to the official PowerShell documentation on [`Types.ps1xml`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_types.ps1xml?view=powershell-6) and [`Format.ps1xml`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_format.ps1xml?view=powershell-6).

... Rest of content... 
```

Reply:
Good suggestion, added